### ISSUE TITLE
Desktop. DropdownMenu. Ripple effect doesn't end if we move a pressed mouse outside of popup

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#841]
Compose 0.5.0-build229

```
import androidx.compose.desktop.Window
import androidx.compose.material.Button
import androidx.compose.material.DropdownMenu
import androidx.compose.material.DropdownMenuItem
import androidx.compose.material.MaterialTheme
import androidx.compose.material.Text
import androidx.compose.runtime.getValue
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.runtime.setValue

fun main() = Window {
    MaterialTheme {
        var expandedState by remember { mutableStateOf(false) }
        Button(onClick = { expandedState = !expandedState }) {
            Text("Menu")
        }
        DropdownMenu(expanded = expandedState, onDismissRequest = { expandedState = false }) {
            DropdownMenuItem({}) {}
        }
    }
}
```

1. Press mouse button down on the item
2. Move mouse outside of the popup
3. Release mouse button

Expected:
Ripple effect ends when we move outside of the item

Actual:
Ripple effect stays

https://user-images.githubusercontent.com/5963351/124123598-28cb1900-da80-11eb-8c16-8fd19fe30e87.mp4

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-rc3

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5375">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
